src/components/BookList.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] As suggested by the previous reviewer, you shouldn't make use of disable eslint, rather pls fix your errors.  I noticed that `BookList.js` is currently generating a `react/prop-types` warning. To fix this, I suggest adding propTypes to your `BookList` component. This will help validate the props passed to the component and prevent runtime errors. Here's an example of how you could add propTypes:

```javascript
BookList.propTypes = {
  books: PropTypes.arrayOf(
    PropTypes.shape({
      id: PropTypes.number.isRequired,
      title: PropTypes.string.isRequired,
    })
  ).isRequired,
};